docs/uptime-guide/security.asciidoc

```diff
@@ -58,3 +67,71 @@ PUT /_security/user/jacknich
 }
 ---------------------------------------------------------------
 // CONSOLE
+
+[float]
+=== Create a role for read-only Uptime users
+
+You'll need a role that lets users who are allowed to access the Heartbeat indices, but are not allowed to adjust Uptime settings. These users must have
+their `application` privileges set to `read`, instead of `all`, as in the
+example below.
+
+You can create this with the following request:
+
+
+["source","sh",subs="attributes,callouts"]
+---------------------------------------------------------------
+PUT /_security/role/uptime_read
+{ 
+  "indices" : [
+    {
+      "names" : [
+        "heartbeat-*"
+      ],
+      "privileges" : [
+        "read",
+        "view_index_metadata"
+      ],
+      "field_security" : {
+        "grant" : [
+          "*"
+        ]
+      },
+      "allow_restricted_indices" : false
+    }
+  ],
+  "applications": [
+    {
+      "application": "uptime",
+      "privileges": [ "read" ],
+      "resources": [ "*" ]
+    }
+  ],
+  "transient_metadata" : {
+    "enabled" : true
+  }
+}
+---------------------------------------------------------------
+// CONSOLE
+
+[float]
+=== Assign the role to a privileged user
+
+Next, you'll need to create a user with both the `uptime` role, and another role with sufficient {kibana-ref}/kibana-privileges.html[Kibana privileges],
+such as the `kibana_admin` role.
+You can do this with the following request:
+
```

Comment:
In this section we link to the Kibana privileges page, but then recommend the `kibana_admin` role, which appears to only be mentioned in the ES reference, [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.6/built-in-roles.html). 

It sounds like we need a step before this one that talks about the Kibana privileges.
What if we offered an eample with the `kibana_admin` role, but then linked to the kibana privileges for more information? Something like:

```
[float]
=== Create a role with Kibana privileges

The Elastic Stack provides a set of fixed, {ref}/built-in-roles.html[built-in roles] that you can assign to users.
One of these roles, `kibana_admin`, can be used to grant access to all features in Kibana.
Alternatively, you can view the {kibana-ref}/kibana-privileges.html[Kibana privileges], and create a custom role with specific Kibana privileges.
```

Then we could move on to this section where we update the example to also apply the `kibana_admin` role.